test/system/lifecycle/lifecycle_test.go

```diff
@@ -280,7 +280,7 @@ func getJWKS(ctx context.Context, client rest.Interface, relativeURI string) ([]
 	return jwksReq.DoRaw(ctx)
 }
 
-func requestAPIServer(ctx context.Context, caBundle []byte, apiserverURL, bearerToken string) (*metav1.Status, error) {
+func requestAPIServer(caBundle []byte, apiserverURL, bearerToken string) (*metav1.Status, error) {
 	req, err := http.NewRequest("GET", apiserverURL, nil)
```

Comment:
You can keep the `ctx` argument and pass it here like 
```suggestion
	req, err := http.NewRequestWithContext(ctx, "GET", apiserverURL, nil)
```